rakefile.rb

```diff
@@ -109,6 +114,52 @@ def download_file(project_name, file_name, uri)
   file
 end
 
+def download_pksim_portable(branch)
+  puts "Downloading PK-Sim portable".light_blue
+  portable_file_name ='pk-sim-portable-setup.zip'
+  appveyor_project_name = 'pk-sim'
+  portable_uri = "https://ci.appveyor.com/api/projects/#{APPVEYOR_ACCOUNT_NAME}/#{appveyor_project_name}/artifacts/#{portable_file_name}?branch=#{branch}"
+
+  download_dir = "C:/projects/ospsuite-r"
+  portable_zip_package = download_file(appveyor_project_name, portable_file_name, portable_uri, download_dir)
+  
+  Utils.run_cmd('7z', %W[e #{portable_zip_package}])
+
+  pksim_minimal_dir = "C:/projects/ospsuite-r/pksim_minimal.zip"
+  command_line = %W[
+    a 
+    #{pksim_minimal_dir} 
+    Castle.Core.dll 
+    Castle.Windsor.dll 
+    DevExpress.Data.v21.2.dll 

```

Comment:
can we use wildcards here (`DevExpress.Data.v*.dll `) in order to avoid adjusting the version in the future?

Reply:
What is this?!? It's not what we discussed @georgeDaskalakis

Reply:
@Yuri05 wildcard added.
@msevestre quick call?

Reply:
I think it's good to document the rationale here so that we can discuss. We said we wanted to upload the zip as a separate package. Instead, we are doing something else. Why

Reply:
But we are uploading as a seperate file no? The only difference here is that we are extracting the PKSim portable to create a new zip that has even less files (PKSim minimal) and that is what we are providing as an artifact.